semver.md

```diff
@@ -286,6 +286,19 @@ modification depends on whether you updated your dependencies in order to fix
 a bug or introduce new functionality. I would usually expect additional code
 for the latter instance, in which case it's obviously a minor level increment.
 
+#### Special note for JVM or similar environments
+
+In JVM or other environments where only one version of a library could 
+be present in a running system, the transitive dependencies effectively become
+part of the public API and thus the version of the user library SHOULD be 
```

Comment:
I would prefer "Interface" over "API", but that issue is still open to some debate.